src/api/__main__.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+"""Code to run if this package is used as a Python module."""
+
+# Third-Party Libraries
+import connexion
+
+# cisagov Libraries
+from api import encoder
+
+
+def main():
+    """Run Main Application.."""
```

Comment:
Get rid of unnecessary caps:
```suggestion
    """Run main application.."""
```